pkgs/development/tools/clang-tools/wrapper

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+buildcpath() {
+  local path
+  while (( $# )); do
+    case $1 in
+        -isystem|-I)
```

Comment:
idirafter could also be used.

Reply:
I just noticed parsing of `-I` was wrong any, since there can be no space in between. Since we only use `-isystem` right now in stdenv, I just simplified the code of the wrapper rather than having a half broken implementation for `-I`. `-Idirafter` is also not used in nixpkgs.